app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -1617,6 +1617,48 @@ class ExplorationActivityTest {
     explorationDataController.stopPlayingExploration(isCompletion = false)
   }
 
+  @Test
+  @RunOn(TestPlatform.ROBOLECTRIC) // TODO(#3858): Enable for Espresso.
+  fun testExpActivity_showHintAndSolution_hasCorrectContentDescription() {
```

Comment:
Just to confirm, does this fail without the fix?

Reply:
Yes @BenHenning, test case is failing without parsing HTML. 
![image](https://user-images.githubusercontent.com/43074241/196857392-84f7d1a9-37ef-41ac-a978-5d9fee1fa1fe.png)

Reply:
Thanks for checking!